src/main/java/io/orkes/conductor/client/http/OrkesEventClient.java

```diff
@@ -67,6 +67,16 @@ public List<EventHandler> getEventHandlers(String event, boolean activeOnly) {
         return eventResourceApi.getEventHandlersForEvent(event, activeOnly);
     }
 
+    @Override
+    public List<EventHandler> getEventHandlers() {
+        return eventResourceApi.getEventHandlers();
+    }
+
+    @Override
+    public void handleIncomingEvent(Map<String, Object> payload) {
+        this.eventResourceApi.handleIncomingEvent(payload);
```

Comment:
Nit: here you're calling it as `this.eventResourceApi` and above just `eventResourceApi`